microsite/data/plugins/devpod.yaml

```diff
@@ -0,0 +1,9 @@
+title: DevPod
+author: terasky.com
+authorUrl: https://github.com/terasky-oss
+category: Development
+description: Automatically launch fully-contained dev environments with DevPod, directly from Backstage!
+documentation: https://github.com/TeraSky-OSS/backstage-plugins/tree/main/plugins/devpod-plugin
+iconUrl: https://devpod.sh/docs/media/devpod-logo.png
```

Comment:
Do you have permission to use this logo?

Reply:
changed the image to use the image from the public OSS GitHub repo

Reply:
Just because it's public doesn't mean we can use it any way we like, that's been my experience. Can you share some documentation from them allowing this use, please?

Reply:
as its under an OSS repo i would think it is allowed, but in any event i opened an issue on the DevPod repo to get clarification from their end

Reply:
The approval has been given in the linked issue from the devpod team at loft